ui/scss/component/_channel.scss

```diff
@@ -287,15 +306,19 @@ $actions-z-index: 2;
       background: transparent;
       .channel__list-item,
       .channel-staked__wrapper {
-        background-color: var(--color-card-background-highlighted);
+        //background-color: var(--color-menu-background-active);
+
+        .claim-preview__title {
+          color: var(--color-menu-icon-active);
+        }
       }
     }
 
     &:first-child {
       .channel__list-item {
-        border-top: 1px solid var(--color-border);
-        border-top-left-radius: var(--border-radius);
-        border-top-right-radius: var(--border-radius);
+        //border-top: 1px solid var(--color-border);
```

Comment:
I don't think everything from &:first-child to the second } is needed and can be removed